### ISSUE TITLE
Property `message` in error response body cannot be accessed

### ISSUE THREAD

**Issue by NikhilNanjappa** [hyperledger/composer#1894]
### Description
Using node I make a typical API POST request for which I have a `.catch` block with response variable say `err`. On error, the response body is returned as such (as seen from the Hyperledger Composer REST app). All this works fine
```
{
  "error": {
    "statusCode": 500,
    "name": "Error",
    "message": "error trying invoke chaincode. Error: chaincode error (status: 500, message: Error: Payment needs to be of positive value)",
    "stack": "Error: error trying invoke chaincode. Error: chaincode error (status: 500, message: Error: Payment needs to be of positive value)\n    at _initializeChannel.then.then.then.then.catch (/home/ubuntu/.nvm/versions/node/v6.11.1/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:839:34)"
  }
}
```
### My Issue

I am simply trying to fetch the `message` property from the above response and send it to the UI. But weirdly `err.message` **does not** give me the value I see inside `message` but returns the entire response body(same with `err.stack`).

So basically -- **`err`, `err.message` and `err.stack` returns the same output** with the only difference of datatype. `err` is of type `object` whereas the other two are of type `string`.

**How do I end up fetching only the value inside a property say `message` ?**

PS:
`err.statusCode` does return the correct value ie. "500".
`err.name` returns "StatusCodeError"(not "Error" that I see in the response)

**Comment by sstone1** [comment]
@NikhilNanjappa if you're using the REST API, then this is dependent on the REST client you're using. I have seen some modules like `request` offer a "simple" mode where the error is collapsed into an Error object and thrown to the user to handle.

If you want to access the "message" property in the JSON response body, you need to turn off this "simple" mode. See https://www.npmjs.com/package/request-promise for more information.
